tests/data/instances/sequence_tagging/verb_semantics_instance_test.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=no-self-use,invalid-name
+from typing import List
+
+from numpy.testing import assert_array_almost_equal
+
+from deep_qa.data.data_indexer import DataIndexer
+from deep_qa.data.instances.instance import TextInstance
+from deep_qa.data.instances.sequence_tagging.pretokenized_tagging_instance import PreTokenizedTaggingInstance
+from deep_qa.data.instances.sequence_tagging.verb_semantics_instance import VerbSemanticsInstance
+from deep_qa.data.tokenizers import tokenizers
+from deep_qa.common.params import Params
+from ....common.test_case import DeepQaTestCase
+
+
+class TestVerbSemanticsInstance(DeepQaTestCase):
+    def setUp(self):
+        super(TestVerbSemanticsInstance, self).setUp()
+
+    def tearDown(self):
+        super(TestVerbSemanticsInstance, self).tearDown()
+
+    def test_read_from_arguments(self):
+        """sentence: List[str], verb: (int, int), entity: (int, int), state_change: str, arg1: (int, int), arg2: (int, int)"""
```

Comment:
I think there's not really a need to have a docstring on a test method like this.  We can see the format from looking at the instance class docstrings.